internal/app/api/connector.go

```diff
@@ -185,7 +185,7 @@ func uninstall[Config models.ConnectorConfigObject](connectorManager *integratio
 func install[Config models.ConnectorConfigObject](connectorManager *integration.ConnectorManager[Config],
 ) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		installed, err := connectorManager.IsInstalled(context.Background())
+		installed, err := connectorManager.IsInstalled(context.TODO())
```

Comment:
Better to use request context to be able to trace calls (sorry i should have said this on previous PR)